src/styles/booksStyles.css

```diff
@@ -0,0 +1,211 @@
+.book-container {
+  margin: 0 6rem 0 6rem;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.single-book {
+  margin: 2rem 2rem 0 0;
+  padding: 0;
+  border: solid 1px rgba(129, 127, 127, 0.322);
+  display: flex;
+  justify-content: space-between;
+}
+
+.book-info {
+  width: 35%;
+  margin: 0 0 0 2rem;
+  padding: 0;
+}
+
+.book-category {
+  margin: 1.5rem 0 0.5rem 0;
+  font-family: "Montserrat", sans-serif;
+  font-weight: 700;
+  color: rgba(129, 127, 127, 0.856);
+}
+
+.book-title {
+  font-family: "Roboto Slab", serif;
+  margin: 0;
+  font-size: 1.8rem;
+  letter-spacing: -0.2px;
+}
+
+.book-author {
+  margin-top: 0;
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #5d93c2a6;
+}
+
+.book-buttons {
+  margin: 1.2rem 0 1rem 0;
+  display: flex;
+}
+
+.single-book-button {
+  margin: 0.3rem 1.7rem 0 0;
+  padding: 0;
+  font-family: "Roboto Slab", serif;
+  background-color: transparent;
+  border: none;
+  color: #0763b37c;
+  font-size: 0.875rem;
+  font-weight: 300;
+  align-items: center;
+  cursor: pointer;
+  position: relative;
+}
+
+.single-book-button:not(:last-child)::after {
+  content: "";
+  position: absolute;
+  top: 50%;
+  right: -1rem;
+  width: 1px;
+  height: 80%;
+  transform: translateY(-50%);
+  background-color: rgba(121, 114, 114, 0.219);
+}
+
+.completed {
+  margin: 0;
+  padding: 0;
```

Comment:
- [ ] You could consider using a reusable class for the `margin` and `padding`property as it reoccurs many times in your codebase as well for CSS best practices

Reply:
Thanks! Created the `.reset` class ✔️